wiki/Tournaments/KEL/4/en.md

```diff
@@ -0,0 +1,222 @@
+---
+tags:
+  - KEL4
+  - KEL LAN
+  - Polish LAN
+---
+
+# KEL LAN Tournament 4
+
+![KEL LAN Tournament 4](img/kel4-banner.jpg)
+
+**KEL LAN Tournament 4** is a 1v1 multi-bracket LAN tournament located in Gdańsk, Poland hosted by ::{ flag=PL }:: [myksmyks](https://osu.ppy.sh/users/6840526), ::{ flag=PL }:: [flapczek](https://osu.ppy.sh/users/8210988) and ::{ flag=DE }:: [TheHunter1](https://osu.ppy.sh/users/6496016). It's the fourth instalment of the KEL LAN Tournament series.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2025-10-17/2026-01-11 |
+| Qualifiers | 2026-01-12/2026-01-26 |
+| Screening phase | 2026-01-26/2026-02-09 |
+| Round of 32, Round of 16, Quarterfinals | 2026-03-14 |
+| Semifinals, Finals, Grand Finals | 2026-03-15 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 40% of the prize pool, unique profile badge |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 25% of the prize pool |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 15% of the prize pool |
+| 4th place | 12% of the prize pool |
+| 5th–6th place | 8% of the prize pool |
+
+![](img/badge-top1.png "KEL LAN Tournament 4 winner badge")
+
+## Organisation
+
+| Position | Member(s) |
+| :-- | :-- |
+| Organisers | ::{ flag=PL }:: [myksmyks](https://osu.ppy.sh/users/6840526), ::{ flag=PL }:: [flapczek](https://osu.ppy.sh/users/8210988), ::{ flag=DE }:: [TheHunter1](https://osu.ppy.sh/users/6496016) |
+| Admins | ::{ flag=PL }:: [Anczak](https://osu.ppy.sh/users/8587966), ::{ flag=PL }:: [Daria Maria](https://osu.ppy.sh/users/10806219), ::{ flag=AT }:: [FCnt](https://osu.ppy.sh/users/7336729) |
+| Production team | ::{ flag=GB }:: [ilw8](https://osu.ppy.sh/users/14167692), ::{ flag=PL }:: [flapczek](https://osu.ppy.sh/users/8210988), ::{ flag=PL }:: [myksmyks](https://osu.ppy.sh/users/6840526) |
+| Developer | ::{ flag=AU }:: [Fairy Bread](https://osu.ppy.sh/users/8306102) |
+| Mappooler | ::{ flag=PL }:: [Bartek22830](https://osu.ppy.sh/users/6404027) |
+| GFX | ::{ flag=US }:: [Spoo](https://osu.ppy.sh/users/11805037), ::{ flag=PL }:: [flapczek](https://osu.ppy.sh/users/8210988), ::{ flag=RU }:: [Ang](https://osu.ppy.sh/users/19124145) |
+| Referees | TBA |
+| Commentators | TBA |
+| Sheeter | ::{ flag=SK }:: [Mavosiik](https://osu.ppy.sh/users/18927594), ::{ flag=PL }:: [P a t r i c k](https://osu.ppy.sh/users/6814521) |
+
+
+## Links
+
+- [Information spreadsheet](https://docs.google.com/spreadsheets/d/17YUODiQXblESi9mciJqn2deFu1TGFKP83k2tkx0W8Ug/)
+- [Announcement post](link)
+- [Discussion thread](link)
+- [Discord server](https://discord.gg/TSgMXsaZRR)
+- [Livestream](https://www.twitch.tv/keltournaments)
+- [YouTube channel](https://www.youtube.com/@KELtournaments)
+- [Tournament bracket](https://challonge.com/kelosu4)
+
+<!-- ## Participants
```

Comment:
just delete and leave out instead of including commented sections

and why is the full mappool here (commented out)? is it copypasted from the previous lan as a placeholder?